### PR TITLE
Crud rewrite

### PR DESCRIPTION
**Pull request by jooohhn**
## Major
- [x] Updated getTableRows() so that it pulls metadata information alongside row values for CRUD robustness
   - [x] metadata contains a row's primary key property and associated value

## Minor
- [x] Updated semantics to be more consistent across all functions/values e.g. getTableValues -> getTableRows
- [x] Reordered methods in file so that they are grouped by similarity e.g. graphql, CRUD, etc.
- [x] Updated tests

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[ignore]
```
Comment (amilajack) [inline .flowconfig]: What does this change do?

File: package.json [hunk 2]
```diff
@@ -61,6 +61,7 @@
     "cassandra-driver": "^3.4.0",
     "cors": "^2.8.4",
     "debug": "^3.1.0",
+    "electron": "^1.8.3",
```
Comment (amilajack) [inline package.json]: We only need this in development. Should be moved to `devDependencies`

**Review by amilajack — DISMISSED** [review]

**Comment by amilajack** [comment]
Make sure to merge master into this branch and remove all the `lib` files. `lib` files were removed in a recent commit

File: .flowconfig (hunk 1 above)
Comment (jooohhn) [reply .flowconfig]: typo, will fix

File: package.json (hunk 2 above)
Comment (jooohhn) [reply package.json]: Will do

**Comment by amilajack** [comment]
What's the status of this PR?

**Pull request merged**